libs/elastic-explorer/src/lib/modules/search/components/elastic-query-editor/elastic-query-editor.component.ts

```diff
@@ -33,4 +35,8 @@ export class ElasticQueryEditorComponent extends OnDestroy$ implements OnInit {
   runQuery(): void {
     this.elasticExplorerService.setElasticQuery(JSON.parse(this.code.value));
   }
+
+  onInitEditor(editor: monaco.editor.IStandaloneCodeEditor): void {
+    this.editorRef.emit(editor);
```

Comment:
To avoid the output/input chaining, I would advise to just set it into the `ElasticExplorerService` (or another dedicated service maybe if that one is getting a bit too big) and handle the expand/collapse in the `FiltersFormComponent` directly 🐱 (and so make use of `[class-my-fullscreen]` instead of `Renderer` for simplicity)